src/composition.rs

```diff
@@ -551,11 +578,32 @@ impl Composition {
                 imports: vec![],
                 eof: vec![],
                 body: vec![ast::Statement::Expr(Box::new(
-                    expr_statement,
+                    expr_statement.to_owned(),
```

Comment:
I'm not sure if it's that big of a deal, but I think I'd prefer a straight `.clone` here. It does the same thing, but the `clone` makes it explicit (and easily greppable). One of these days, we'll probably have to reduce `clone` calls, so that will make it explicit.